GetStream/stream-chat-swift
# [CIS-1353] Reactions cover text in message bubble

### 🔗 Issue Link
CIS-1353

### 🎯 Goal

Reactions shouldn't cover the text in the message bubble

### 🛠 Implementation

Follow design by offsetting the reactions bubble view by 5px from the center X.

<img width="115" alt="Screenshot 2021-12-06 at 17 18 15" src="https://user-images.githubusercontent.com/4924709/144882932-6ece98ef-65ee-4f15-ba43-3be38cb67ee7.png">

### 🧪 Testing

1. Add multiple reactions to the message
2. Verify the reactions don't cover the text.

### 🎨 Changes

Adjusting the correspondent layout constraint.

### ☑️ Checklist
